app/api/patient/upload/after/route.ts

```
@@ -16,14 +16,54 @@ export async function POST(request: Request) {
+    if (!imageId) {
+      return new Response(
+        JSON.stringify({ message: 'Image ID is required' }),
+        {
+          status: 400,
+          headers: { 'Content-Type': 'application/json' },
+        }
+      );
+    }
+
+    // check if image id is a number
+    if (isNaN(Number(imageId))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)